### PR TITLE
Update to support SDK 2.0

### PR DESCRIPTION
**Pull request by bennettp123**
Hi there,

I'm sending you this pull request because I forked your tea_timer app and updated it for SDK 2.0. I'm sending this pull request in case you wanted to merge it into your own codebase.

Note that I've also made a few other changes:
- Larger font
- Swapped up and down buttons (up->reduce time, down->increase time; this matches the next/prev behaviour in builtin music app)
- Add ability to stop & start timer
- Snap timer changes to nearest 30 seconds
- Changed default time to three minutes

I've included these in the pull request, but if you want any of them removed, let me know and I'll revert them and resubmit the pull request.

Alternatively, feel free to ignore me and I won't bother you again. :)

Thanks in advance,
Bennett.

### CODE REVIEW

**Comment by mfollett** [comment]
Thanks, I haven't looked at SDK 2.0 yet. I'll look at this by the end of the weekend and merge it.

File: appinfo.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "versionLabel": "1.0",
+    "uuid": "081b46ac-5530-413c-b82e-a192fb698838",
+    "appKeys": {},
+    "longName": "Tea Timer",
+    "versionCode": 1,
+    "capabilities": [
+        ""
+    ],
+    "shortName": "tea_timer",
+    "companyName": "bennettp123",
```
Comment (mfollett) [inline appinfo.json]: Could you make the companyName just an empty string?

File: src/display.c [hunk 2]
```diff
@@ -1,24 +1,29 @@
+//#include <time.h> //already defined in <pebble.h>!
```
Comment (mfollett) [inline src/display.c]: Could you remove the commented out code in this file? here

File: src/display.c [hunk 3]
```diff
@@ -32,27 +37,40 @@ void alert() {
...
+  	window_set_background_color(window, GColorBlack);
 
-  layer_add_child(&window.layer, &timer.layer);
-
-  text_layer_set_text(&timer, "Starting...");
+	timer = text_layer_create(GRect(14, 49, 130, 50));
+	text_layer_set_text_color(timer, GColorWhite);
+	text_layer_set_background_color(timer, GColorClear);
+	//text_layer_set_text_alignment(&timer, GTextAlignmentCenter);
+	//text_layer_set_font(&timer, fonts_get_system_font(FONT_KEY_GOTHAM_30_BLACK));
```
Comment (mfollett) [inline src/display.c]: here

File: appinfo.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
+{
+    "versionLabel": "1.0",
+    "uuid": "081b46ac-5530-413c-b82e-a192fb698838",
+    "appKeys": {},
+    "longName": "Tea Timer",
+    "versionCode": 128408,
```
Comment (mfollett) [inline appinfo.json]: This causes a build error. Can you set this to a lower number (1 is fine, or whatever)?

Comment (bennettp123) [inline appinfo.json]: Done.

File: appinfo.json (hunk 1 above)
Comment (bennettp123) [inline appinfo.json]: Done

**Comment by mfollett** [comment]
This is awesome, thanks for updating this.

I realized I don't have a license file in the code. Do you have any objections with this being licensed [BSD 3-clause](http://www.tldrlegal.com/license/bsd-3-clause-license-%28revised%29)?

**Comment by bennettp123** [comment]
Urgh I'm so new to github. Sorry about the close & reopen! I'm learning heaps though. I now know why the recommended workflow is branch-commit-merge! :)

No problems about the update; I found tea_timer it a few months ago and have been using once or twice a day ever since; so naturally, it was the first thing I updated when upgrading to 2.0. :)

I have no objections to the BSD 3-clause license.

**Comment by mfollett** [comment]
Great, thanks again for your help!

**Pull request merged**